x-pack/plugins/session_view/public/components/process_tree/helpers.ts

```diff
@@ -4,9 +4,40 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { Process, ProcessEvent, ProcessMap } from '../../../common/types/process_tree';
+import {
+  AlertStatusEventEntityIdMap,
+  Process,
+  ProcessEvent,
+  ProcessMap,
+} from '../../../common/types/process_tree';
 import { ProcessImpl } from './hooks';
 
+// if given event is an alert, and it exist in updatedAlertsStatus, update the alert's status
+// with the updated status value in updatedAlertsStatus Map
+export const updateAlertEventStatus = (
+  events: ProcessEvent[],
+  updatedAlertsStatus: AlertStatusEventEntityIdMap
+) =>
+  events.map((event) => {
+    // do nothing if event is not an alert
+    if (!event.kibana) {
```

Comment:
Can do in a follow up PR, but in the future `event.kibana` may have other sub-properties, so may be best to use `event.kibana.alert` to be explicit

Reply:
Thanks Michael, will update in my follow up PR!